_posts/2020-02-22-gsoc2020.md

```diff
@@ -0,0 +1,113 @@
+| 2 | [Command Line Interface (CLI)]({{ site.github.wiki }}/Project-Ideas-2020#project-2-command-line-interface-cli) | easy | create a new package with MDAnalysis-based command line tools | Python, bash, Python packages, CI, MD | @PicoCentauri @orbeckst @fiona-naughton
+| 3 | [Interoperability with RDKit]({{ site.github.wiki }}/Project-Ideas-2020#project-3-interoperability-with-rdkit) | medium | add capability to MDAnalysis to use the RDKit API to convert data structures between MDAnalysis and RDKit | MDAnalysis, RDKit, Python, C++ (?) | @richardjgowers @IAlibay
+| 4 | [Improved atom selections]({{ site.github.wiki }}/Project-Ideas-2020#project-4-improved-atom-selections) | medium | replace the selection parsing code with a more flexible parser | Python | @orbeckst, @IAlibay, @fiona-naughton
+| 5 | [Serialize Universes for parallel computation]({{ site.github.wiki }}/Project-Ideas-2020#project-5-serialize-universes-for-parallel-computation) | medium/challenging | make the central `Universe` data structure serializable with Pickle to enable simple parallelization with Dask or MPI | Python, MDAnalysis I/O, task-based parallelization | @richardjgowers, @orbeckst, @IAlibay, @fiona-naughton 
+| 6 | [Implement TNG support]({{ site.github.wiki }}/Project-Ideas-2020#project-6-implement-tng-support) | challenging | write Cython bindings to the TNG library and write classes to bring this information into MDAnalysis | Cython, C++ | @richardjgowers
+
+# Information for Students
+
+You must meet our [own requirements] if you want to be a student with MDAnalysis
+this year (read all the docs behind these links!). You must also meet the
+[eligibility criteria].
+
+The MDAnalysis community values diversity and is committed to
+providing a productive, harassment-free to every member. Our [Code of
+Conduct] explains the values that we as a community uphold. *Every
+member (and every GSoC student) agrees to follow the [Code of
+Conduct]*.
+
+
+As a start to get familiar with MDAnalysis and open source development you
+should follow these steps:
+
+## Complete the Quick Start Guide
+
+We have a [Quick Start Guide] explaining the basics of MDAnalysis. You
+should go through it at least once to understand how MDAnalysis is
+used. Continue reading the [User Guide] to learn more.
+
+## Introduce yourself to us
+
+Introduce yourself on the [mailing list]. Tell us what you plan to work
+on during the summer or what you have already done with MDAnalysis.
+
+## Close an issue of MDAnalysis
+
+You must have *at least one commit in the development branch of
+MDAnalysis* in order to be eligible, i.e. you must demonstrate that
+you have been seriously engaged with the MDAnalysis project.
+We have a list of [easy bugs][] and suggested [GSOC Starter
+issues][GSOC Starter] to work on in our issue tracker on GitHub. We
+also appreciate if you write more tests or update/improve our
+documentation.
+
+We recommend that you don't leave this until the last minute to allow
```

Comment:
That you start with this before you dive deep into your project. It will help to get to know MDAnalysis and understand how the proposed ideas fit into MDAnalysis.